src/main/java/org/terasology/navgraph/BaseRegion.java

```diff
@@ -20,30 +20,52 @@
 
 /**
  * @author synopia
+ * Abstract class which floors and regions derive from
  */
 public abstract class BaseRegion<N extends BaseRegion> {
     public int id;
     public Set<N> neighborRegions = new HashSet<N>();
     protected BitMap map;
     protected Floor floor;
 
+    /**
+     * Creates a BaseRegion
```

Comment:
To add more value to this docstring you can try to answer a few of the common questions that come to your mind when looking at a method or constructor like this:

- what is the id used for? 
- can I use the same id for two different regions?
- what properties does the freshly created region have? what does it mean for a region to be empty?